docker-for-mac/edge-release-notes.md

```diff
@@ -58,6 +77,7 @@ Fixed an issue that caused VMs running on older hardware with macOS Catalina to
 - Improved the navigation in **Settings** and **Troubleshoot** UI.
 - Fixed a bug in the UEFI boot menu that sometimes caused Docker Desktop to hang during restart. Fixes [docker/for-mac#2655](https://github.com/docker/for-mac/issues/2655) and [docker/for-mac#3921](https://github.com/docker/for-mac/issues/3921).
 - Docker Desktop now allows users to access the host’s SSH agent inside containers. Fixes [docker/for-mac#410](https://github.com/docker/for-mac/issues/410)
+- Docker Machine is not included in the Docker Desktop installer anymore. You can download it separately from the [Docker Machine releases](https://github.com/docker/machine/releases) page.
```

Comment:
Suggest: Docker Machine is no longer included in the Docker Desktop installer.

Reply:
Fixed.